pkg/experiment/symbolizer/symbolizer.go

```diff
@@ -41,6 +41,10 @@ type Config struct {
 	PersistentDebugInfoStore    DebugInfoStoreConfig `yaml:"persistent_debuginfo_store"`
 }
 
+type Limits interface {
+	SymbolizerEnabled(string) bool
+}
+
```

Comment:
I think we can optimize it: let's extend the `Limits` interface defined by frontend.

You may find it interesting:
 * https://go.dev/wiki/CodeReviewComments#interfaces
 * https://100go.co/#interface-on-the-producer-side-6

```suggestion
```